### PR TITLE
Updating with latest code changes done manually

### PR DESCRIPTION
**Pull request by dartion5**

### CODE REVIEW

File: sites/all/themes/scholarly_lite/page.tpl.php.bak [hunk 1]
```diff
@@ -0,0 +1,426 @@
+<?php if (theme_get_setting('scrolltop_display')): ?>
```
Comment (shaunmuscat) [inline sites/all/themes/scholarly_lite/page.tpl.php.bak]: Is there a specific reason why this backup file has been included?

File: sites/all/themes/scholarly_lite/style.css [hunk 2]
```diff
@@ -253,7 +259,7 @@ h1.title { margin-top:0; margin-bottom: 20px; }
 
 #highlighted + #main-content { padding: 20px 0; }
 
-#main-content { padding: 60px 0; }
+#main-content { padding: 0px 0; }
```
Comment (shaunmuscat) [inline sites/all/themes/scholarly_lite/style.css]: This doesn't need to be changed, but for future reference in CSS you don't have to explicitly state `0px` you can just use `0` instead. It isn't much of a difference, but it is good to keep a consistent style with the rest of the file where possible (e.g. `padding: 0 0;` or even just `padding: 0;`).

**Review by shaunmuscat — CHANGES_REQUESTED** [review]
I noticed there is a .bak file in this pull request. If it is there by accident then it should be removed.

**Comment by dartion5** [comment]
I've removed unintended `.bak` file and also updated `style.css` file to keep it consistent with other `.css` files 👍

**Review by shaunmuscat — APPROVED** [review]

**Pull request merged**